src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ConsentOrderNotApprovedDocumentService.java

```diff
@@ -48,11 +48,11 @@ public List<BulkPrintDocument> prepareApplicantLetterPack(FinremCaseDetails case
 
         List<BulkPrintDocument> documents = new ArrayList<>();
 
-        documents.add(coverLetter(caseDetails, authorisationToken));
+        documents.add(notApprovedCoverLetter(caseDetails, authorisationToken, APPLICANT));
         addEitherNotApprovedOrderOrGeneralOrderIfApplicable(caseDetails, documents, authorisationToken);
 
         return documents.size() == 1
-            ? emptyList()  // if only cover letter then print nothing
```

Comment:
That did seem like a useful comment to me.

Reply:
Added back [e180735](https://github.com/hmcts/finrem-case-orchestration-service/pull/1732/commits/e1807355486a1c68b9f813c423ea8033bb9d520d)